paparazzi-gradle-plugin/src/test/projects/application-plugin/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+  ~ Copyright (C) 2018 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<manifest package="app.cash.paparazzi.plugin.test">
+</manifest>
```

Comment:
delete; covered by `namespace` in build.gradle

Reply:
Empty AndroidManifest is required in application/dynamic-feature module, otherwise there is an error

`Caused by: com.android.builder.errors.EvalIssueException: Manifest file does not exist: /paparazzi-gradle-plugin/src/test/projects/supports-application-modules/src/main/AndroidManifest.xml`